avocado/plugins/list.py

```diff
@@ -230,9 +246,20 @@ def configure(self, parser):
                                  parser=parser,
                                  long_arg='--write-recipes-to-directory')
 
+        help_msg = 'Writes output to a json file.'
+        settings.register_option(section='list',
+                                 key='write_to_json_file',
+                                 default=None,
+                                 metavar='JSON_FILE',
+                                 help_msg=help_msg,
+                                 parser=parser,
+                                 long_arg='--write-to-json')
```

Comment:
we already have `--json FILE` arguments across the code. I would go with that to keep the pattern.

Reply:
`--json` it was introduced by the `plugins/jsonresult.py` and it is about `job.run.results`. Assuming that plugins should be independent, IMO we should choose different command line options names to avoid confusion.  IDK, but having `--json` that generates one output with the `run` and a similar `--json` that provides a different output with `list` commands, seems strange to me. Also `--json` is too generic, IMO.

What do you think?

Reply:
I like consistency. Usually when I use a command and I want to check if there is some kind of JSON output I look for the `-j` or `--json` argument, just like in the `ip` command. Having two different options in the same software to write the output to JSON would make me really mad. I know the plugins are different, but they will do the same when using the argument, write to a JSON file.

Reply:
Ok, I don't like much, but I understand your point. I will change on a v2.

Reply:
While `--write-to-json` keeps it similar to the existing `--write-recipes-to-directory`, I do think that `--json` is more broadly used on tools. So I agree with @willianrampazzo